app/scripts/modules/core/src/entityTag/alerts/EntityNotifications.tsx

```diff
@@ -0,0 +1,124 @@
+import { module } from 'angular';
+
+import { react2angular } from 'react2angular';
+import * as React from 'react';
+import { IEntityTags, IEntityTag } from 'core/domain';
+import { NotificationsPopover } from './NotificationsPopover';
+import { Placement } from 'core/presentation/Placement';
+import { Application } from 'core/application';
+import { noop } from 'core/utils';
+
+interface IProps {
+  entity: any;
+  application: Application;
+  placement?: Placement;
+  // a horizontal offset percent string
+  offset?: string;
+
+  className?: string;
+
+  pageLocation: string;
+  entityType: string;
+
+  onUpdate?(): void;
+}
+
+/**
+ * A notifications popover for alerts and notices.
+ * Shows the notifications for a single entity (not rolled up, and not grouped by message)
+ */
+export class EntityNotifications extends React.Component<IProps, void> {
+  public static defaultProps: Partial<IProps> = {
+    placement: 'bottom',
+    offset: '50%',
+    className: '',
+    onUpdate: noop,
+  };
+
+  public render() {
+    const { entity, application, placement, offset, pageLocation, className, entityType, onUpdate } = this.props;
+    const entityTags: IEntityTags = entity.entityTags;
+
+    function alertAnalyticsLabel(): string {
+      const { account, region, entityId } = entityTags.entityRef;
+      const alertsStr = entityTags.alerts.map((tag: IEntityTag) => tag.name).join(',');
+      return [ pageLocation, entityType, account, region, entityId, region, alertsStr ].join(':');
+    }
+
+    function noticeAnalyticsLabel(): string {
+      const { account, region, entityId } = entityTags.entityRef;
+      const noticesStr = entityTags.notices.map((tag: IEntityTag) => tag.name).join(',');
+      return [ pageLocation, entityType, account, region, entityId, noticesStr ].join(':');
+    }
+
+    const tags = entityTags ? [entityTags] : [];
+
+    return (
+      <div className="entity-notifications">
+
+        <NotificationsPopover
```

Comment:
Render alerts for the entity.  Show categories, but do not attempt to group/dedupe identical messages